libs/seiscomp/gui/datamodel/pickerview.cpp

```diff
@@ -4971,6 +4971,7 @@ bool PickerView::addTheoreticalArrivals(RecordViewItem* item,
 		double delta, az1, az2;
 		double elat = SC_D.origin->latitude();
 		double elon = SC_D.origin->longitude();
+		double edep = SC_D.origin->depth();
```

Comment:
This is optional and can throw an exception.

Reply:
Ah! I was not aware of that. I will fix this